interfaces/IBF-dashboard/src/assets/i18n/en.json

```diff
@@ -298,10 +299,10 @@
         "ETH": "Elderly: vulnerable population group Ethiopia: High Resolution Population Density Maps + Demographic Estimates <a href=\"https://data.humdata.org/dataset/ethiopia-high-resolution-population-density-maps-demographic-estimates\">https://data.humdata.org/dataset/ethiopia-high-resolution-population-density-maps-demographic-estimates</a>"
       },
       "small_ruminants": {
-        "ZWE": "Small ruminants exists of the summarised number sheep & goats livestock numbers multiplied with the Livestock unit (LSU): 0.1 as reference unit to aggregate livestock from various species.<br /><br />Source link Zimbabwe:<ul><li>Number of small ruminants (sheep and goats) mentioned within the 2nd round crop- and livestock assessment report 2020/2021 season. Published: 21st of April 2021.<br/> Source assessment:  <a href=\"https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0\">https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0</a></li></ul>"
+        "ZWE": "Small ruminants exists of the summarised number sheep & goats livestock numbers multiplied with the Livestock unit (LSU): 0.1 as reference unit to aggregate livestock from various species.<br /><br />Source link Zimbabwe:<ul><li>Number of small ruminants (sheep and goats) mentioned within the 2nd round crop- and livestock assessment report 2020/2021 season. Published: 21st of April 2021.<li>Source assessment:<br /><a href=\"https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0\">https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0</a></li></ul>"
       },
       "cattle": {
-        "ZWE": "Livestock numbers cattle exists of the number of cattle multiplied with the Livestock unit (LSU): 1.0  as reference unit to aggregate livestock from various species.<br /><br />Source Links Zimbabwe:<ul><li>Number of cattle mentioned within the 2nd round crop- and livestock assessment report 2020/2021 season. Published: 21st of April 2021. Source assessment: <a href=\"https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0\">https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0</a>.</li></ul>"
+        "ZWE": "Livestock numbers cattle exists of the number of cattle multiplied with the Livestock unit (LSU): 1.0  as reference unit to aggregate livestock from various species.<br /><br />Source Links Zimbabwe:<ul><li>Number of cattle mentioned within the 2nd round crop- and livestock assessment report 2020/2021 season. Published: 21st of April 2021.<li>Source assessment:<br /><a href=\"https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0\">https://fscluster.org/zimbabwe/document/second-round-crop-and-livestock-0</a>.</li></ul>"
```

Comment:
@SwatiTh aren't you forgetting an additional closing </li> here? Because now you have more opening <li>'s then closing </li>'s. Did you check how this looks when you open the popup?